Infrastructure/Music_Portal.Infrastructure.Data/ArtistRepository.cs

```diff
@@ -0,0 +1,65 @@
+﻿using System.Collections.Generic;
+using System.Linq;
+using Music_Portal.Domain.Core;
+using Music_Portal.Domain.Interfaces;
+
+namespace Music_Portal.Infrastructure.Data
+{
+    public class ArtistRepository : IArtistRepository
+    {
+        private readonly ApplicationContext _db;
+
+        public ArtistRepository(ApplicationContext context)
+        {
+            _db = context;
+        }
+
+        public IEnumerable<Artist> GetArtists()
+        {
+            if (!_db.Artists.Any())
+            {
+                return new List<Artist>();
```

Comment:
I don't think we need to create List here, much simpler to use `Array.Empty<T>()`